File: 01/part1.pl
#!/usr/bin/perl
while(<>) {
 chomp;
 push @a,$_;
}
for $x (@a) {
 for $y (@a) {
  if($x + $y == 2020) {
   die("$x $y ",$x*$y)
  }
 }
}

File: 01/part1.py
#!/usr/bin/python3
import sys
a = sys.stdin.read().strip()

for x in a.split("\n"):
	for y in a.split("\n"):
  		if( int(x)+int(y) == 2020):
  			print(int(x)*int(y))
  			exit()
